snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -127,7 +127,8 @@ private void assertDescription(String term, EntityProxy.Concept nameType, Entity
             Latest<SemanticEntityVersion> latestDescriptionSemantic = stampCalc.latest(semanticVersion);
             if (latestDescriptionSemantic.isPresent()) {
                 Component descriptionType = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_TYPE, latestDescriptionSemantic.get());
-                if (PublicId.equals(descriptionType.publicId(), nameType)) {
+                Component caseSensitivity = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE, latestDescriptionSemantic.get());
+                if (PublicId.equals(descriptionType.publicId(), nameType) && PublicId.equals(caseSensitivity.publicId(), caseSensitive)) {
```

Comment:
This looks good. The only thing you need now is an assertion at the end of the method after the loop. Essentially you want to do something like: assertEquals(term, actual.get())

Reply:
Code added.